pootle/apps/pootle_store/updater.py

```diff
@@ -153,6 +153,12 @@ def should_update_source(self):
                 and self.unit.source != self.newunit.source)
 
     @cached_property
+    def resurrected(self):
+        return (
+            self.newunit
+            and self.unit.isobsolete() != self.newunit.isobsolete())
```

Comment:
this condition will be `True` for opposite change as well. Probs we need to be explicit here or just rename the method.